### PR TITLE
Feature/mouse touch support

### PR DESCRIPTION
**Pull request by nvh**
Fixes https://github.com/motif/company/issues/5015

### CODE REVIEW

File: framer/Events.coffee [hunk 1]
```diff
@@ -25,15 +25,14 @@ Events.enableEmulatedTouchEvents = (enable=true) ->
 		Events.TouchStart = "touchstart"
 		Events.TouchEnd = "touchend"
 		Events.TouchMove = "touchmove"
+	Events.Click = Events.TouchEnd
```
Comment (eelco) [inline framer/Events.coffee]: Tab vs Space :)

File: framer/Events.coffee [hunk 2]
```diff
@@ -25,15 +25,14 @@ Events.enableEmulatedTouchEvents = (enable=true) ->
...
 
 Events.enableEmulatedTouchEvents(false)
 
-# Let's make sure the touch events work on desktop too
-if not Utils.isTouch()
+# Let's make sure the touch events work with a mouse too
+if Utils.isMouse()
```
Comment (eelco) [inline framer/Events.coffee]: This seems like it still should be `not Utils.isTouch()`? (only enable the emulator if there’s no touch)

**Comment by nvh** [comment]
Only didn't add the `isMouse()` function to the `isTouch` checks in https://github.com/koenbok/Framer/blob/master/framer/GestureInputRecognizer.coffee, but I think that doesn't make sense

File: framer/Events.coffee (hunk 2 above)
Comment (eelco) [inline framer/Events.coffee]: I now see there’s an extra guard in the enable. Maybe this `if` should be removed and make the logic for enabling live purely there?

File: framer/Utils.coffee [hunk 3]
```diff
@@ -353,6 +353,11 @@ Utils.isTouch = ->
 	window.ontouchmove is null and
 	window.ontouchend is null
 
+Utils.isMouse = ->
+	window.onmousedown is null and
+	window.onmouseup is null and
+	window.onmousemove is null
+
```
Comment (eelco) [inline framer/Utils.coffee]: Does this work? I’d expect for backwards compatibility this would still return `true` on iOS?

Comment (nvh) [inline framer/Utils.coffee]: You're right. Stupid oversight

**Pull request closed without merging**